src/Stats.re

```diff
@@ -80,12 +83,17 @@ let make =
 
   let isWide = MaterialUi.useMediaQuery("(min-width: 600px)");
   <>
-    {switch (resultsQuery) {
-     | Loading => <CircularProgress />
-     | NoData
-     | Error(_) => <span> {text("Error")} </span>
-     | Data(data) =>
-       let results = data##results |> toRecord;
+    {switch (resultsQuery, settingsQuery) {
+     | (_, Loading)
+     | (Loading, _) => <CircularProgress />
+     | (NoData, _)
+     | (_, NoData)
+     | (Error(_), _)
+     | (_, Error(_)) => <span> {text("Error")} </span>

```

Comment:
Why does `NoData` also result in error text?

Reply:
`NoData` is before it starts to load?
Created an issue to handle it and Errors in a uniform way throuhhout the application: https://github.com/Yakimych/result-log/issues/74

Reply:
Yes, it goes from `NoData` to `Loading` to `Error`/`Success`, so I don't think `NoData` should be treated as error 😄 But yeah, it is fine to fix it in another issue.